biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -1637,96 +1637,94 @@ subroutine fire_litter_fluxes(currentSite, currentPatch, &
              ! across ground surface. 
              ! -----------------------------------------------------------------------
              
-             sapw_m   = currentCohort%prt%GetState(sapw_organ, element_id)
-             struct_m = currentCohort%prt%GetState(struct_organ, element_id)
-             leaf_m   = currentCohort%prt%GetState(leaf_organ, element_id)
-             fnrt_m   = currentCohort%prt%GetState(fnrt_organ, element_id)
-             store_m  = currentCohort%prt%GetState(store_organ, element_id)
-             repro_m  = currentCohort%prt%GetState(repro_organ, element_id)
-             
-             ! Absolute number of dead trees being transfered in with the donated area
-             num_dead_trees = (currentCohort%fire_mort*currentCohort%n * &
+			if ( prt_params%woody(pft) == itrue) then !trees only, May require additional accounting for the atm_fluxes when PFT is grass. 
+													  ! Goal was to not add to pools in the event of grass fire. Grass would typically not introduce a flux of dead leaves	
```

Comment:
I would like to see an else attached to this if clause where we use a logging message to indicate non-woody disturbance is not supported here, and include an endrun() and a verbose message. This prevents vulnerable code in case someone modifies disturbance generation to allow for non-woody, they will then realize it needs to be applied here as well.